### DISCUSSION TITLE
Frontend setup issue in local with UPYOG UI- Module not found: Can't resolve '@egovernments/digit-ui-libraries'

### DISCUSSION THREAD

**Discussion by rasasmal2** [upyog/UPYOG#538] [Q&A]
After successful yarn install, while running yarn start getting  Failed to compile error.
I am getting error in "UPYOG\frontend\upyog-ui\web\micro-ui-internals\example\src" path.
Screenshot is attached here:
<img width="782" alt="Error Capture" src="https://github.com/upyog/UPYOG/assets/124992537/7f37360e-aca6-4829-b7ab-a3e1cf730a9f">

Module not found: Can't resolve '@egovernments/digit-ui-libraries' in 'C:\Users\RS\Projects\dev\UPYOG\frontend\upyog-ui\web\micro-ui-internals\example\src'

**Comment by support-upyog** [comment]
Please join the call for discussion.
